### PR TITLE
fix(wasm-runtime): preserve full bigint values in fs proxy codec

### LINKED ISSUE #3225: wasm-runtime/fs-proxy silently truncates standalone BigInt values to signed 64-bit
## Steps to reproduce

Run this from repo root

```js
import * as memfs from 'memfs'
import { createFsProxy, createOnMessage } from './wasm-runtime/fs-proxy.js'

const fs = memfs.createFsFromVolume(memfs.Volume.fromJSON({}))
fs.__echo__ = (x) => x

const onMessage = createOnMessage(fs)
globalThis.postMessage = (data) => onMessage({ data })

const proxy = createFsProxy(memfs)

const standalone = 2n ** 63n
const wrapped = { value: 2n ** 63n }

console.log(String(proxy.__echo__(standalone)))
console.log(String(proxy.__echo__(wrapped).value))
```

#### Observed behavior

```console
-9223372036854775808
9223372036854775808
```

#### Expected behavior

Should it be?
```console
9223372036854775808
9223372036854775808
```

If the intent is to support only signed 64-bit `bigint` values, this should throw on overflow instead of silently truncating.

### PR DESCRIPTION
**Pull request by trivikr**
Fixes: https://github.com/napi-rs/napi-rs/issues/3225

## Summary by CodeRabbit

* **Bug Fixes**
  * More reliable BigInt serialization/decoding and clearer handling of payload-overflow errors.
  * Improved error-path behavior to ensure errors are encoded and reported safely.

* **Chores**
  * Increased in-memory payload capacity to support larger message payloads.

* **Tests**
  * Expanded tests for BigInt boundary values and added a case to verify unsupported-value error reporting.

### CODE REVIEW

File: wasm-runtime/fs-proxy.js [hunk 1]
```diff
@@ -89,9 +89,7 @@ const encodeValue = (memfs, value, type) => {
       return view
     }
     case 9: {
-      const view = new BigInt64Array(1)
-      view[0] = value
-      return new Uint8Array(view.buffer)
+      return new TextEncoder().encode(String(value))
```
[the remark this answers is not in the document]
Comment (trivikr) [reply wasm-runtime/fs-proxy.js:107]: Fixed in https://github.com/napi-rs/napi-rs/pull/3233/changes/e54a99a80c49cd344af01108ea676e5a1a4b4540

**Comment by Brooooooklyn** [comment]
@codex review
